website/content/docs/agent/index.mdx

```diff
@@ -361,6 +361,12 @@ To run Vault Agent:
    $ vault agent -h
    ```
 
+As with Vault, the `-config` flag can be used in three different ways:
+
+- Use the flag once to name the path to a single specific configuration file.
+- Use the flag multiple times to name multiple configuration files, which will be composed at runtime.
```

Comment:
I don't normally see `composed` used in this sense, maybe it's a British-ism?  I would use merged or combined instead.

Reply:
I actually took this wording from one of our existing learn docs: https://developer.hashicorp.com/vault/tutorials/operations/configure-vault

Reply:
I don't think that's proper English either.  You can say "this cake is composed of the following ingredients", but I've never seen it used in the form "I composed these ingredients to make a cake".  My English grammar isn't so hot but I think it comes down to transitive vs intransitive use. @tjperry07 wdyt?

Reply:
For the record, it's definitely a grammatical form I recognise and the sentence does make 100% sense to me, so it could be a British English thing maybe? To give a maths/tech example, it's used in this way regularly when talking about composing functions (e.g. for a composition `f . g`, f and g are composed).

I don't really have qualms with changing it, but we should probably update it in both places if we do.

Reply:
Composed can be used, but it's reading strange to Americans because we usually use it when talking about art and feelings. She composed the painting beautifully. They composed themselves before walking into the room. 

In this case merged, combined, even compiled (if appropriate) is better. Just because we follow US English and all US English quirks. But composed is grammatically correct.